lib/open_api_spex/test/test_assertions.ex

```diff
@@ -12,7 +12,7 @@ defmodule OpenApiSpex.TestAssertions do
   Asserts that `value` conforms to the schema with title `schema_title` in `api_spec`.
   """
   @spec assert_schema(map, String.t(), OpenApi.t()) :: map | no_return
-  def assert_schema(value = %{}, schema_title, api_spec = %OpenApi{}) do
```

Comment:
Is the type spec incorrect now? Should it be `@spec assert_schema(term, ...` ?

Reply:
True, thanks for the catch! I will update it